crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -608,11 +627,20 @@ impl MeshUniform {
             material_and_lightmap_bind_group_slot: u32::from(material_bind_group_slot)
                 | ((lightmap_bind_group_slot as u32) << 16),
             tag: tag.unwrap_or(0),
-            pad: 0,
+            aabb_center: aabb.map(|a| a.center().into()).unwrap_or(Vec3::ZERO),
+            aabb_half_extents: aabb.map(|a| a.half_size().into()).unwrap_or(Vec3::ZERO),
+            uv0_range: uv_range_to_vec4(uv0_range),
+            uv1_range: uv_range_to_vec4(uv1_range),
         }
     }
 }
 
+fn uv_range_to_vec4(range: Option<Aabb2d>) -> Vec4 {
+    range
+        .map(|r| Vec4::new(r.min.x, r.min.y, r.max.x, r.max.y))
```

Comment:
```suggestion
        .map(|r| Vec4::new(r.min.x, r.min.y, r.max.x - r.min.x, r.max.y - r.min.y))
```

Could optimise the shader slightly by moving the subtract here?